packages/esm-appointments-app/src/appointments/scheduled/appointments-list.component.tsx

```diff
@@ -1,38 +1,44 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { filterByServiceType } from '../utils';
 import { useAppointmentList } from '../../hooks/useAppointmentList';
 import AppointmentsTable from '../common-components/appointments-table.component';
 
 interface AppointmentsListProps {
-  appointmentServiceType?: string[];
+  appointmentServiceTypes?: Array<string>;
+  date: string;
+  excludeCancelledAppointments?: boolean;
   status?: string;
   title: string;
-  date: string;
-  filterCancelled?: boolean;
 }
+
 const AppointmentsList: React.FC<AppointmentsListProps> = ({
-  appointmentServiceType,
+  appointmentServiceTypes,
+  date,
+  excludeCancelledAppointments = false,
   status,
   title,
-  date,
-  filterCancelled = false,
 }) => {
   const { appointmentList, isLoading } = useAppointmentList(status, date);
 
-  const appointments = filterByServiceType(appointmentList, appointmentServiceType).map((appointment) => ({
-    id: appointment.uuid,
-    ...appointment,
-  }));
+  const appointmentsFilteredByServiceType = filterByServiceType(appointmentList, appointmentServiceTypes).map(
+    (appointment) => ({
+      id: appointment.uuid,
+      ...appointment,
+    }),
+  );
```

Comment:
Question (purely for me to learn how to use `useMemo` better than just putting it everywhere xD): 
I'm wondering if wrapping `appointmentsFilteredByServiceType` in its own useMemo would help. My concern is that if `appointmentsFilteredByServiceType` recalculates on every render (creating a new array reference each time), it would trigger `activeAppointments` to recalculate unnecessarily.
But if the props `appointmentList` and `appointmentServiceTypes` change on every render anyway, is there any benefit to using `useMemo` for `appointmentsFilteredByServiceType`?